libs/mark-flow-ui/src/components/candidate_contest.test.tsx

```diff
@@ -249,9 +249,12 @@ describe('supports write-in candidates', () => {
     userEvent.click(modal.getByText('Accept'));
     expect(screen.queryByRole('alertdialog')).not.toBeInTheDocument();
 
-    expect(updateVote).toHaveBeenCalledWith(candidateContestWithWriteIns.id, [
-      { id: 'write-in-lizardPeople', isWriteIn: true, name: 'LIZARD PEOPLE' },
-    ]);
+    expect(updateVote).toHaveBeenCalledWith(
+      candidateContestWithWriteIns.id,
+      [{ id: 'write-in-lizardPeople', isWriteIn: true, name: 'LIZARD PEOPLE' }],
+      // expect.objectContaining({ nativeEvent: expect.any(Object) })
```

Comment:
Is this a TODO, or should we just delete it?

Reply:
https://github.com/votingworks/vxsuite/pull/4284/commits/f977e7e976ffbf77a2e882c749ee8352e1fe3ffe

Updated to use `syntheticEventMatcher`